core/src/main/java/io/github/kaktushose/jdac/dispatching/context/internal/RichInvocationContext.java

```diff
@@ -16,6 +16,11 @@ public record RichInvocationContext(
         Runtime runtime
 
 ) {
+
```

Comment:
i dont like the naming, especially when used in a static import it is not clear what is meant